external/fv3fit/fv3fit/reservoir/transformers/transformer.py

```diff
@@ -29,6 +30,11 @@ def decode(self, x: ArrayLike) -> Sequence[ArrayLike]:
         pass
 
 
+class Transformer(BaseTransformer, Reloadable):
+    def __init__(self, **kwargs):
+        self.super().__init__(**kwargs)
+
+
```

Comment:
`Reloadable` is now redundant in the `DoNothingAutoencoder` subclass instance and can be removed.